openproblems/tasks/denoising/methods/dca.py

```diff
@@ -14,7 +14,7 @@ def _dca(adata):
     # by casting list(range(dim(adata.obsm['train'][1]) as header for vars and the next index [2] for header of obs
     adata2 = sc.AnnData(adata.obsm["train"])
     Y = dca(adata2, threads=1)
-    adata.obsm["denoised"] = Y.X #Y.X should call the count matrix of DCA.
+    adata.obsm["denoised"] = Y.X  # Y.X should call the count matrix of DCA.
```

Comment:
This won't work. Per `https://github.com/theislab/dca/blob/master/dca/api.py#L211`, `Y == None`. I think you want `adata2.X`